Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -584,7 +586,7 @@ public struct Data : Equatable, Hashable, RandomAccessCollection, MutableCollect
         @usableFromInline typealias Buffer = (UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8,
                                               UInt8, UInt8, UInt8, UInt8, UInt8, UInt8) //len  //enum
         @usableFromInline var bytes: Buffer
-#elseif arch(i386) || arch(arm) || arch(arm64_32)
+#elseif arch(i386) || arch(arm) || arch(arm64_32) || arch(wasm32)
```

Comment:
Apologies if this is a silly question, but is there also a `wasm64` architecture that we need to worry about adding to the if condition above for 64 bit? (this makes me wonder if we should switch these conditionals over to using `_pointerBitWidth(_32)`/`_pointerBitWidth(_64)` for consistency with newer platforms but that doesn't need to be part of this change)

Reply:
I checked the history of this code piece but I didn't see any reason not to use `_pointerBitWidth` here!